android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -385,15 +385,18 @@ public void onClick(View v) {
         showSummarySection();
         mBtnTip.setEnabled(false);
 
-        String monthName = (String) android.text.format.DateFormat.format("MMM", new Date());
+        String monthName = (String) android.text.format.DateFormat.format("MMMM", new Date());
+        String monthNameShort = (String) android.text.format.DateFormat.format("MMM", new Date());
         int lastDate = Calendar.getInstance().getActualMaximum(Calendar.DAY_OF_MONTH);
         int currentYear = Calendar.getInstance().get(Calendar.YEAR);
 
-        String adsMonthlyStatement = new StringBuilder(monthName)
-                                             .append(" 1 - ")
-                                             .append(monthName)
+        String adsMonthlyStatement = new StringBuilder("1")
                                              .append(" ")
+                                             .append(monthNameShort.replaceAll("\\.", ""))
```

Comment:
Just for confirmation
There is no period in Rewards Panel in my Device.
Dec 1 - Dec 31 currently showing in Rewards panel.
Now it's updated to 1 Dec - 31 Dec.

Reply:
That's correct @sujitacharya2005 we needed to match date format with desktop